src/Table/Footer.jsx

```diff
@@ -120,10 +129,13 @@ Footer.propTypes = {
   onPageChange: PropTypes.func,
   numColumns: PropTypes.number.isRequired,
   numPages: PropTypes.number.isRequired,
+  showLastPage: PropTypes.bool,
+  isLoading: PropTypes.bool,
 };
 
 Footer.defaultProps = {
   onPageChange: () => {},
+  showLastPage: true,
```

Comment:
No good reason besides that that was the previous behavior - if the tables not lazy we want to show the last page.